### PR TITLE
[FIX] fetchmail should not set as 'Seen' messages which raised exception

### PR DESCRIPTION
**Pull request by eLBati**
upstream: https://github.com/odoo/odoo/pull/5705

### CODE REVIEW

File: addons/fetchmail/fetchmail.py [hunk 1]
```diff
@@ -206,12 +206,13 @@ def fetch_mail(self, cr, uid, ids, context=None):
                                                                  save_original=server.original,
                                                                  strip_attachments=(not server.attach),
                                                                  context=context)
+                            imap_server.store(num, '+FLAGS', '\\Seen')
                         except Exception:
                             _logger.exception('Failed to process mail from %s server %s.', server.type, server.name)
                             failed += 1
+                            imap_server.store(num, '-FLAGS', '\\Seen')
```
Comment (pedrobaeza) [inline addons/fetchmail/fetchmail.py]: Is this needed? What happened if you don't put it? Doesn't it remain unseen?

Comment (eLBati) [inline addons/fetchmail/fetchmail.py]: > Is this needed? What happened if you don't put it? Doesn't it remain
> unseen?
> 
> I agree: it is not needed

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by yvaucher** [comment]
:+1:

**Comment by eLBati** [comment]
Merge?

**Comment by yvaucher** [comment]
Merged

**Pull request merged**